public/js/services/game.js

```diff
@@ -55,152 +54,184 @@ angular.module('mean.system')
       $timeout(decrementTime, 950);
     };
 
-    socket.on('id', function(data) {
-      game.id = data.id;
-    });
+    socket.on('id', (data) => {
+            game.id = data.id;
+        });
 
-    socket.on('prepareGame', function(data) {
-      game.playerMinLimit = data.playerMinLimit;
-      game.playerMaxLimit = data.playerMaxLimit;
-      game.pointLimit = data.pointLimit;
-      game.timeLimits = data.timeLimits;
-    });
+    socket.on('prepareGame', (data) => {
+            game.playerMinLimit = data.playerMinLimit;
+            game.playerMaxLimit = data.playerMaxLimit;
+            game.pointLimit = data.pointLimit;
+            game.timeLimits = data.timeLimits;
+        });
 
-    socket.on('gameUpdate', function(data) {
+    socket.on('gameUpdate', (data) => {
 
-    // Update gameID field only if it changed.
-    // That way, we don't trigger the $scope.$watch too often
-      if (game.gameID !== data.gameID) {
-        game.gameID = data.gameID;
-      }
+            // Update gameID field only if it changed.
```

Comment:
Fix the spacing from here to end of file based on the earlier comments I added at the beginning of the file. Some lines have extra spaces while others are ok